_layouts/page.html

```diff
@@ -11,17 +13,15 @@
     {% include header.html %}
   </head>
   <body class="workshop">
-
     {% include github-ribbon.html %}
-
```

Comment:
On Sat, Apr 18, 2015 at 04:29:35PM -0700, Greg Wilson wrote:
> >      {% include github-ribbon.html %}
> > -
> 
> Seemed more consistent with the lack of blank lines elsewhere in the
> template.

I agree that they should go, I just don't think removing them is part
of “Adding a 'root' variable to allow pages in sub-directories”.  It's
not a big deal to have a bit of leakage in commit atomicity, but it is
nice to do right ;).